CIGI-588: Top bar styling fixes

johnbxu opened the pull request:
- increased menu item font size to 14px
- removed centre alignment of expanded menu content on larger screens
- aligned expanded menu items to cigi logo on the left side

#### Description of changes
resolves CIGIHub/cigi-tickets#588

gabrieltanchen reviewed (changes requested):
The search dropdown styling needs to be fixed.

![Screen Shot 2021-05-07 at 2 29 23 PM](https://user-images.githubusercontent.com/10100465/117493198-a8d17880-af40-11eb-8526-67d21ee12d3d.png)

johnbxu commented:
Fixed search popup.

gabrieltanchen reviewed (changes requested):
Is the menu item and the logo suppose to line up?

![Screen Shot 2021-05-10 at 10 00 42 AM](https://user-images.githubusercontent.com/10100465/117671481-d43da880-b176-11eb-8bbe-3df526f72ce6.png)

johnbxu commented:
Hid scroll bar for larger screens and adjusted padding so menu items either line up with the logo, or are indented slightly. Also  added focus on search input after dropdown button is clicked.
